<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.xp.test.http.ddt.DataProviderByCSV" tests="1" time="0.000" ignored="1" failures="0" timestamp="08 十二月 2020 13:46:07 GMT" skipped="0" hostname="DESKTOP-TJK6KB0" errors="1">
  <testcase name="testDataDrivenByCSVFile1" time="0.000" classname="com.xp.test.http.ddt.DataProviderByCSV">
    <error type="org.testng.TestNGException" message="
Data Provider static java.util.List com.xp.test.http.ddt.DataProviderByCSV.getTestData1() throws java.io.IOException must return either Object[][] or Iterator&lt;Object&gt;[], not interface java.util.List">
      <![CDATA[org.testng.TestNGException: 
Data Provider static java.util.List com.xp.test.http.ddt.DataProviderByCSV.getTestData1() throws java.io.IOException must return either Object[][] or Iterator<Object>[], not interface java.util.List
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:133)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1020)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1110)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- testDataDrivenByCSVFile1 -->
  <testcase name="testDataDrivenByCSVFile" classname="com.xp.test.http.ddt.DataProviderByCSV">
    <skipped/>
  </testcase> <!-- testDataDrivenByCSVFile -->
</testsuite> <!-- com.xp.test.http.ddt.DataProviderByCSV -->
